### PR TITLE
Add WCD, GCDI, GCDM, WIPG, WIPL summary output for Field and Region

### PR DESCRIPTION
**Pull request by totto82**
These are summary output that is used for CO2STORE simulations.

### CODE REVIEW

**Comment by totto82** [comment]
jenkins build this opm-simulators=4477 please

File: src/opm/output/eclipse/Summary.cpp [hunk 1]
```diff
@@ -2230,7 +2230,11 @@ static const std::unordered_map< std::string, Opm::UnitSystem::measure> single_v
   {"FGIPL"    , Opm::UnitSystem::measure::gas_surface_volume },
   {"FGIPG"    , Opm::UnitSystem::measure::gas_surface_volume },
   {"FPR"      , Opm::UnitSystem::measure::pressure },
-
+  {"FGCDI"    , Opm::UnitSystem::measure::liquid_surface_volume },
+  {"FGCDM"    , Opm::UnitSystem::measure::liquid_surface_volume },
+  {"FWCD"     , Opm::UnitSystem::measure::gas_surface_volume },
```
Comment (bska) [inline src/opm/output/eclipse/Summary.cpp:2235]: This appears to use two different units of measurement for CO2, depending on whether it is in the gas phase (`FG*`) or in the water phase (`FW*`).  Is that intentional?

File: src/opm/output/eclipse/Summary.cpp [hunk 2]
```diff
@@ -2230,7 +2230,11 @@ static const std::unordered_map< std::string, Opm::UnitSystem::measure> single_v
...
+  {"FWIPG"    , Opm::UnitSystem::measure::liquid_surface_volume },
+  {"FWIPL"    , Opm::UnitSystem::measure::liquid_surface_volume }
```
Comment (bska) [inline src/opm/output/eclipse/Summary.cpp:2237]: Tiny nit, but could you put a trailing comma on the last entry here please?  That way, it's slightly easier to add new entries when we need to.

**Review by bska — COMMENTED** [review]
This looks good for the most part.  I don't quite understand why CO2 volumes are reported in different units of measurement depending on the associate phase, but I also admit that I haven't looked too closely at this application.

File: src/opm/output/eclipse/Summary.cpp (hunk 1 above)
Comment (totto82) [reply src/opm/output/eclipse/Summary.cpp]: Thanks for noticing.

**Comment by totto82** [comment]
jenkins build this opm-simulators=4477 please

**Comment by totto82** [comment]
image

**Comment by totto82** [comment]
Seems to work as expected.  Results are from modified CO2STORE test case. https://github.com/OPM/opm-tests/pull/910

**Review by bska — APPROVED** [review]
Thanks a lot for the updates.  I'll merge into master.

**Pull request merged**